### DISCUSSION TITLE
建议参考国科大模板增加windows和linux的编译脚本

### DISCUSSION THREAD

**Discussion by Nick-Hopps** [sikouhjw/gdutthesis#38] [General]

**Comment by sikouhjw** [comment]
不添加脚本的原因有几个：
1. 限制了编译顺序，有时不需要编译参考文献或没有生成参考文献，却用了 `bibtex`/`biber` 会报错
2. `latexmk` 在大多数情况下工作正常
3. 配置 `latexmkrc` 我觉得也没什么必要，需要删除辅助文件时，用 `latexmk -C` 也只会剩下 `.bbl` 和 `.tec`，没太大的必要
3. 平时用 `xelatex` 即可，生成终稿／查看完整效果再用 `latexmk -xelatex`

**Comment by sikouhjw** [comment]
在 `texdoc latexmk` 的 `bibtex_use [1]` 选项中有这么一段话
> Note that the value 1.5 does not work properly if the document uses biber instead of bibtex.
(There’s a long story why not.)

`1.5` 是最优解，但不支持 `biber`，所以配置 `latexmk` 是没什么必要的事，无非就
```
% 编译
latexmk -xelatex main
% 删除几乎所有衍生文件
latexmk -C main
% 比 `-C` 少删除一些
latexmk -c main
```
这几行，所以配置的意义不大。
